package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "enrise-logger",
   "description": "Logger used within Enrise projects and module's",
-  "version": "0.2.1",
+  "version": "1.0.0",
```

Comment:
Yes I know, my experience is that before the 1.0 release you can decide on any breaking change. But since we're already using it in production let's move to 1.0 then.